docs/root/intro/arch_overview/health_checking.rst

```diff
@@ -19,7 +19,10 @@ unhealthy, successes required before marking a host healthy, etc.):
   considered healthy. Envoy also supports connect only L3/L4 health checking.
 * **Redis**: Envoy will send a Redis PING command and expect a PONG response. The upstream Redis
   server can respond with anything other than PONG to cause an immediate active health check
-  failure.
+  failure. Optionally, Envoy can perform EXISTS on a user-specified key. If the key does not exist
+  it is considered a passing healthcheck. This allows the user to mark a Redis instance for
```

Comment:
If the key does exist it would also be a passing health check, right?

Reply:
If the key exists, Redis returns 1 and we will fail the healthcheck.

Reply:
Ah, OK, I misread the next part.